### DISCUSSION TITLE
Which data structures does Objeck currently lack?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#375] [General]
The Fusion language supports `List`, `Queue`, `Stack`, `HashSet`, `SortedSet`, `Dictionary` and `SortedDictionary`. It seems Objeck has all of them except the sorted versions. Please correct me if I'm wrong. Thank you.

**Comment by objeck** [comment]
In Objeck, the ``Dictionary`` class is a sorted ``Map`` of key/value pairs. Also, the ``ComapreVector`` class supports sorting.

**Reply by ghost** [reply]
There is no unsorted versions? Btw, what about `Set`?

**Comment by objeck** [comment]
In Java, a ``Set`` is an unordered collection like a ``Vector`` or ``List``. Again, my time is limited, and I tend only to implement new library classes if required. Adding **more** to Objeck means more testing and maintenance.
